api/provision/csp-create-portfolio.ts

```diff
@@ -5,21 +5,17 @@ import errorLogger from "@middy/error-logger";
 import inputOutputLogger from "@middy/input-output-logger";
 import validator from "@middy/validator";
 import { Context } from "aws-lambda";
-import JSONErrorHandlerMiddleware from "middy-middleware-json-error-handler";
-import {
-  NewPortfolioPayload,
-  ProvisionCspResponse,
-  ProvisionRequest,
-  provisionRequestSchema,
-} from "../../models/provisioning-jobs";
+import jsonErrorHandlerMiddleware from "middy-middleware-json-error-handler";
+import { HothProvisionRequest, NewPortfolioPayload } from "../../models/provisioning-jobs";
 import { logger } from "../../utils/logging";
```

Comment:
I don't think `NewEnvironmentPayload` is needed as an import.
```suggestion
import { NewPortfolioPayload, HothProvisionRequest } from "../../models/provisioning-jobs";
```